### PR TITLE
Warmer chart colors with stroke

### PR DESCRIPTION
**Pull request by fiskus**
Screenshot from 2021-09-08 08-45-25

### CODE REVIEW

**Review by nl0 — APPROVED** [review]
looks nice, tho on hover i'd make the whole area opaque and the same color as the stroke, but i guess this is just a matter of preference

File: catalog/app/components/StackedAreaChart/StackedAreaChart.js [hunk 1]
```diff
@@ -133,22 +141,36 @@ export default function MultiSparkline({
...
+            // eslint-disable-next-line react/no-array-index-key
+            key={`area:${i}`}
+            onMouseEnter={handleAreaEnter(i)}
+            onMouseLeave={handleAreaLeave}
+          >
+            <polygon
+              points={SVG.pointsToSVG(area)}
+              fill={
+                // If color is #COLOR7
+                areaPaints[i].ref.length === 7
```
Comment (nl0) [inline catalog/app/components/StackedAreaChart/StackedAreaChart.js:155]: i dont get it. what's special about color 7?

Comment (fiskus) [reply catalog/app/components/StackedAreaChart/StackedAreaChart.js:155]:
```
#COLORX
1234567
```

Comment (fiskus) [reply catalog/app/components/StackedAreaChart/StackedAreaChart.js:155]: #ABCDEF - hex color is a string 7 characters long

Comment (fiskus) [reply catalog/app/components/StackedAreaChart/StackedAreaChart.js:155]: Another variant is `url(#key)`, and it throws error when passed to `lighten()` function

Comment (nl0) [reply catalog/app/components/StackedAreaChart/StackedAreaChart.js:155]: oh i see

**Comment by nl0** [comment]
i've spotted a bug -- mouse tracking is a little broken: try hovering over the areas somewhere on the left side of the chart -- area highlighting is a little off (i've checked staging and it's ok there)

**Comment by nl0** [comment]
it's because of how polylines work, here's the explanation (i set polylines' fill opacity to non-zero):
Screenshot 2021-09-08 at 18 48 55

**Pull request merged**